src/test/charts/functest/templates/ingress-EKS.yaml

```diff
@@ -16,17 +16,19 @@ metadata:
     "nginx.ingress.kubernetes.io/proxy-send-timeout": "60"
 spec:
   rules:
-    - host: {{ printf "%s.%s" .Values.productReleaseName .Values.ingressDomain }}
+    {{- range $ingressName := .Values.ingressNames }}
+    - host: {{ printf "%s.%s" . $.Values.ingressDomain }}
```

Comment:
should be using `$ingressName` instead of `.` here, for consstency

Reply:
Absolutely. I spotted this in `ingress-KITT.yaml` last night, but I missed this one.